### PR TITLE
HighlightScreen: Pending

### PR DESCRIPTION
**Pull request by raynesmike**
Im still trying to find out how to modify the color to match the App Requirement

### CODE REVIEW

**Comment by raynesmike** [comment]
highlighter1

**Comment by Dulaya** [comment]
Have you looked at this simulator: https://react-syntax-highlighter.github.io/react-syntax-highlighter/demo/prism.html

I also did some research and couldn't find a better library. I looked into the React Syntax Highlighter code base, and I don't think this problem can be solved using the React Native Syntax Highlighter library alone. I feel like it's NOT possible to highlight the codes according to Figma without "over engineering" it by making some bizarre modifications. I know Lasha wanted you to use the library, but I just don't see that working out. In my opinion, it'd be easier just to write your own rules from scratch than to try to configure the library to highlight the syntax according to Figma.

File: src/components/highlighter/HighLighter.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import SyntaxHighlighter from 'react-native-syntax-highlighter';
+// import {docco} from 'react-syntax-highlighter/styles/hljs';
```
Comment (lasharela) [inline src/components/highlighter/HighLighter.js:3]: Remove

**Review by lasharela — APPROVED** [review]

**Review by Dulaya — APPROVED** [review]
It works for me when I tried it.

**Review by Nap1400 — APPROVED** [review]

**Pull request merged**